requirements-dev.txt

```diff
@@ -1,5 +1,6 @@
 behave==1.2.6
-behaving==1.5.6
+behaving==2.0.0
+Appium-Python-Client<=0.52
```

Comment:
what is appium python client doing here?

Reply:
We need to lock it to an old version in order to upgrade `behaving` on Python 2.